Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
+  dev_server:
+    cmds:
+      - >
```

Comment:
In the `dev_server` command, consider adding error handling logic to gracefully handle failure cases, such as when the build command fails, instead of letting it exit silently or crashing the entire process.